tests/test_base_workflow.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from dsc.exceptions import InvalidDSpaceMetadataError
+from dsc.item_submission import ItemSubmission
+
+
+def test_base_workflow_item_submission_iter(base_workflow_instance):
+    assert next(base_workflow_instance.item_submissions_iter()) == ItemSubmission(
+        dspace_metadata={
+            "metadata": [
+                {"key": "dc.title", "value": "Title", "language": "en_US"},
+                {"key": "dc.contributor", "value": "Author 1", "language": None},
+                {"key": "dc.contributor", "value": "Author 2", "language": None},
+            ]
+        },
+        bitstream_uris=[
+            "s3://dsc/workflow/folder/123_01.pdf",
+            "s3://dsc/workflow/folder/123_02.pdf",
+        ],
+        metadata_keyname="workflow/folder/123_metadata.json",
+        metadata_uri="",
+    )
+
+
+def test_base_workflow_create_dspace_metadata(
+    base_workflow_instance,
+    item_metadata,
+):
+    assert base_workflow_instance.create_dspace_metadata(item_metadata) == {
+        "metadata": [
+            {"key": "dc.title", "language": "en_US", "value": "Title"},
+            {"key": "dc.contributor", "language": None, "value": "Author 1"},
+            {"key": "dc.contributor", "language": None, "value": "Author 2"},
+        ]
+    }
```

Comment:
Hmm, I'm curious how you determine whether to add "_success" or "raises_exception" / "raise_error" to the test name? 🤔 

I brought this up with @ghukill while we were working on tests in `timdex-dataset-api`. I think some consistency or setting a standard per project would be nice! For instance...

* Given that [this test](https://github.com/MITLibraries/dspace-submission-composer/pull/21/files#diff-4aef579184063b7807d85afdb351da1571f6b608b597e746a3045eca07437f1aR4) is a single method with a single test, I'd expect [this test](https://github.com/MITLibraries/dspace-submission-composer/pull/21/files#diff-f8a31e340d9b850865bff75a6a3f13de37ab5cdcfc67ea2a2f9086332ea880d3R25) and [this other test](https://github.com/MITLibraries/dspace-submission-composer/pull/21/files#diff-f8a31e340d9b850865bff75a6a3f13de37ab5cdcfc67ea2a2f9086332ea880d3R25) should also include "_success".

Reply:
You are correct, good catch!